backend-stub/setup/index.js

```diff
@@ -0,0 +1,107 @@
+const fetch = require("node-fetch");
+const uniqid = require("uniqid");

```

Comment:
What is the difference between this azure function and the api call in backend-stub/index.js?

Reply:
The api call in backend-stub/index.js was really just to test things locally so I didn't have to wait on the Azure function to build each time. The difference is that Azure handles async functions and the request results a bit differently so I had to adjust that. I will eventually delete the /test endpoint after the Azure is fully working.

Reply:
Ok got it, makes sense. We should probably wait to merge until after we delete that?